src/proto/NEAR.proto

```diff
@@ -0,0 +1,97 @@
+syntax = "proto3";
+
+package TW.NEAR.Proto;
+option java_package = "wallet.core.jni.proto";
+
+/// Provides container for unsigned 128 bit integers.
+message Uint128 {
+    bytes number = 1;
+}
+
+message PublicKey {
+    uint32 key_type = 1;
```

Comment:
what's key_type?

Reply:
we can have keys with different types of signing algorithms, e.g. ETH-compatible